.jenkins/pytorch/win-build.sh

```diff
@@ -126,10 +126,8 @@ set DISTUTILS_USE_SDK=1
 set CMAKE_GENERATOR=Ninja
 
 if not "%USE_CUDA%"=="1" (
-  if "%REBUILD%"=="" (
-    set NO_CUDA=1
-    python setup.py install
-  )
+  if "%REBUILD%"=="" set NO_CUDA=1
```

Comment:
It's a bit unclear why we set `NO_CUDA=1` only if `REBUILD` is not empty. Ideally it should have nothing to do with it.

Reply:
@yf225 we do the same thing in the other branch (see line 147 on the left/145 on the right). If you can explain to me why it's correct there I can explain to you why it's correct here :)

Reply:
It's likely not correct in either cases, feel free to remove them if local build passes :)